example4.py

```diff
@@ -1,24 +1,29 @@
-# Corrected Python code
-
 from math import pi
 
-def calculate_area(radius):
+def calculate_area(radius: float) -> float:
+    if radius < 0:
+        raise ValueError("Radius cannot be negative")
     area = pi * radius ** 2
     return area
```

Comment:
Using type hints for the parameters and return type in `calculate_volume` is recommended.